version.txt

```diff
@@ -1 +1 @@
-os0.2.0
+7.2.5
```

Comment:
This file actually gets modified in the `make ansible` command from docker-splunk, but it's good to update this too :P